bundle/manifests/aws-load-balancer-operator.clusterserviceversion.yaml

```diff
@@ -424,6 +416,8 @@ spec:
                 env:
                 - name: AWS_SHARED_CREDENTIALS_FILE
                   value: /etc/aws-credentials/credentials
+                - name: OPERAND_IMAGE
```

Comment:
It makes sense to prefix this variable with `RELATED_IMAGE_` as this will allow us to automatically populate the CSV's `spec.relatedImages` field at CPaaS build time, see [OSBS doc](https://osbs.readthedocs.io/en/latest/users.html#creating-the-relatedimages-section).
This, in turn, will help to enable one of the [main requirements](https://docs.openshift.com/container-platform/4.10/operators/operator_sdk/osdk-generating-csvs.html#olm-enabling-operator-for-restricted-network_osdk-generating-csvs) for the disconnected mode.